rollup.config.js

```diff
@@ -33,6 +38,7 @@ export default {
     alias({
       resolve: ['/index.js', '/index.jsx', 'js', 'jsx'] //optional, by default this will just look for .js files or folders
     }),
+    json(),
```

Comment:
Why not use a variable instead of a json file ?
```
const tooltipMessages = {
"name" : "message",
...
}
```

Reply:
I thought it might be better in case we start adding translation in futures 🤔. Do let me know  if the content should be moved to a variable.

Reply:
@n1tin Thats a great idea, could you open an issue about this ?

Reply:
Sure. Will do that.